docker-compose.yml

```diff
@@ -21,8 +21,8 @@ services:
       - CLIENT_PROTOCOL=http
       - CLIENT_DOMAIN=localhost
       - CLIENT_PORT=3000
-      - CLIENT_HOSTNAME=localhost:3000
-      - CLIENT_ADDRESS=http://localhost:3000
+      - CLIENT_HOSTNAME
+      - CLIENT_ADDRESS
```

Comment:
I would try leaving `docker-compose.yml` as-is, and make the `CLIENT_HOSTNAME` and `CLIENT_ADDRESS` changes in `docker-compose.tailscale.yml`. That should hopefully fix the failing integration test.